.env.runtime.schema

```
@@ -0,0 +1,33 @@
+#
+# @defaultSensitive=false
+# @import(./.env.azure.schema, enabled=eq($AUTH_MODE, "azure"))
+# ---
+
+# @optional @type=port
+API_PORT=8000
+
+# API base URL
+# @public @type=url
+NUXT_API_URL=if(
+    eq($APP_MODE, dev),
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The eq($APP_MODE, dev), key has incorrect delimiter

(IncorrectDelimiter)